### DISCUSSION TITLE
[Q] Nvidia GPU power consumption

### DISCUSSION THREAD

**Discussion by ryosoftware** [blakeblackshear/frigate#8136] [General]
Hello,

I'm using Frigate and also HA and other services in dockers and my CPU is a Xeon E5-2560-v2 with 32GB of RAM and I'm pretty happy on how CPU works and about the average of CPU usage.

Recently I've buy a Nvidia 1660TI card (due to Xeon processors lacks from an integrated GPU) and I'm not using it in conjunction with Frigate, but I'm thinking on configure the system to allow Frigate to use the GPU card, but I'm a little worried with the power consumption (GPU power consumption will be until 120W).

Do you have any statistic about power consumption related to Frigate using some modern GPUs?

Thanks

**Comment by NickM-27** [comment]
I can say that using my nvidia 3050 with frigate and running a compreface face recognition model the power consumption is ~ 30 W

**Reply by ryosoftware** [reply]
Wow, this isn't too much. I assume that the instructions in https://docs.frigate.video/configuration/hardware_acceleration/#nvidia-gpus are correct and I need to install the Nvidia container toolkit.
Do you refer to 30W is the consumption difference BTW use frigate without GPU activation and with the Nvidia 3050, not the total consumption, no?

**Reply by NickM-27** [reply]
The 30W is simply what the GPU is reporting as total power usage

![Screen Shot 2023-10-12 at 07 54 31 AM](https://github.com/blakeblackshear/frigate/assets/14866235/71fb8656-bb3b-4c26-9de5-3051bb61cac0)

**Reply by ryosoftware** [reply]
Hmm, sounds good. What app do you use to get this info? Thanks

**Reply by NickM-27** [reply]
it is part of the unraid dashboard, using the nvidia-smi api

**Reply by ryosoftware** [reply]
Can you confirm if instructions at https://docs.frigate.video/configuration/hardware_acceleration/#nvidia-gpus are in date and I need to install the Nvidia container toolkit? Thanks!

**Reply by NickM-27** [reply]
Yes this is required

**Reply by ryosoftware** [reply]
Is working, thanks

![imagen](https://github.com/blakeblackshear/frigate/assets/31618496/caf300e5-3a40-42e9-bc5d-16c48b442736)

![imagen](https://github.com/blakeblackshear/frigate/assets/31618496/358b77dc-9e79-4aa0-a016-15756f3800a5)
